### PR TITLE
fix: Corrected performance glue synchronous issue

### PR DESCRIPTION
**Pull request by Abhishk123**

### CODE REVIEW

**Review by ibrahimshuail — APPROVED** [review]

**Review by neeradanelxsi — APPROVED** [review]

**Review by jbigel — DISMISSED** [review]

File: Docs/Performance_Metrics.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# Performance Metrics
+
+### Background
+
```
Comment (neeradanelxsi) [inline Docs/Performance_Metrics.md:4]: you can remove the readme 
https://etwiki.sys.comcast.net/display/FIR/Ripple+Performance+Testing there is already a link with all details. just confirm if we have all the upto date info

Comment (Abhishk123) [reply Docs/Performance_Metrics.md]: removed readme file

**Review by neeradanelxsi — APPROVED** [review]

**Pull request merged**